nss-day-cohort-39/tabloid-full-stack-del_rios
# User Profile

### Changes
- Added userprofil.js, user.js, and userprofiledetails.js
- added userprofile to header only to show for admins

### Testing
- log in as non admin to confirm that the option to click user profiles is not available
- log in as admin to confirm user profiles link is in the header
- view userprofile list
-click details for user and confirm information displaying is accurate for that user
-click return to list
